plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/m2e/core/ui/preferences/MavenErrorPreferencePage.java

```diff
@@ -0,0 +1,130 @@
+/******************************************************************************* 
+ * Copyright (c) 2016 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation 
+ ******************************************************************************/
+package org.jboss.reddeer.eclipse.m2e.core.ui.preferences;
+
+import org.jboss.reddeer.common.logging.Logger;
+import org.jboss.reddeer.jface.preference.PreferencePage;
+import org.jboss.reddeer.swt.impl.combo.LabeledCombo;
+
+/**
+ * Class representing "Maven->Errors/Warnings" preference page.
+ * 
+ * @author jkopriva, ldimaggi
+ *
+ */
+
+public class MavenErrorPreferencePage extends PreferencePage {
+
+	private static final String GROUP_ID = "\"groupId\" duplicate of parent groupId";
+	private static final String VERSION = "\"version\" duplicate of parent version";
+	private static final String PROJECT_CONFIG = "Out-of-date project configuration";
+	private static final String PLUGIN_EXECUTION = "Plugin execution not covered by lifecycle configuration";
+	private static final String MANAGED_VERSION = "Overriding managed version";
+	private static final Logger log = Logger.getLogger(MavenErrorPreferencePage.class);
+
+	/**
+	 * Construct the preference page with "Maven" > "Errors/Warnings".
+	 */
+	public MavenErrorPreferencePage() {
+		super(new String[] { "Maven", "Errors/Warnings" });
+	}
+	
+	/**
+	 * Getter method for "groupId duplicate of parent groupId" LabeledCombo
+	 * @return String value of selected item in labeled combo
```

Comment:
I would change description to something like "Gets message severity level of "groupId duplicate of parent groupId"". If there would be an enum, it could be also reference in description as "Gets message {@link Severity} level of "groupId duplicate of parent groupId"".